jwst/tweakreg/tweakreg_step.py

```diff
@@ -279,35 +279,30 @@ def process(self, input):
             g = grp_img[0]
             if len(g) == 0:
                 raise AssertionError("Logical error in the pipeline code.")
-            group_name = _common_name(g)
             imcats = list(map(self._imodel2wcsim, g))
             # Remove the attached catalogs
             for model in g:
                 del model.catalog
-            self.log.info("* Images in GROUP '{}':".format(group_name))
+            self.log.info(f"* Images in GROUP '{model.meta['group_id']}':")
             for im in imcats:
-                im.meta['group_id'] = group_name
-                self.log.info("     {}".format(im.meta['name']))
+                self.log.info(f"     {im.meta['name']}")
 
             self.log.info('')
 
         elif len(grp_img) > 1:
             # create a list of WCS-Catalog-Images Info and/or their Groups:
             imcats = []
-            all_group_names = []
             for g in grp_img:
                 if len(g) == 0:
                     raise AssertionError("Logical error in the pipeline code.")
                 else:
-                    group_name = _common_name(g, all_group_names)
                     wcsimlist = list(map(self._imodel2wcsim, g))
                     # Remove the attached catalogs
                     for model in g:
                         del model.catalog
-                    self.log.info("* Images in GROUP '{}':".format(group_name))
+                    self.log.info(f"* Images in GROUP '{model.meta['group_id']}':")
```

Comment:
```suggestion
                    self.log.info(f"* Images in GROUP '{model.meta.group_id}':")
```
Same as above.

Reply:
yes, it is a mistake. but I think I would like to use the value actually being passed to `tweakwcs` so I will make a slight change.